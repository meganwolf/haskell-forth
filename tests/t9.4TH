3 1 + STR SALE  TODAY  STR CONCAT3 .
1 1 + STR 2 2 + STR CONCAT2 CR .
19 10 - STR 5 7 / STR 2 2 * STR CONCAT3 CR .