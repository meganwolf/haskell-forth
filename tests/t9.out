4SALETODAY
24
904